modules/claims_api/app/models/claims_api/form_526.rb

```diff
@@ -38,23 +40,29 @@ class Form526
     end
 
     def initialize(params = {})
-      @attributes = []
-      params.each do |name, value|
-        @attributes << name.to_sym
-        begin
-          send("#{name}=", value)
-        rescue StandardError
+      sanitized_params = sanitize_fields(params)
+      sanitized_params.each do |name, value|
+        send("#{name}=", value)
+      end
+    end
+
+    def sanitize_fields(params)
+      bad_fields = params.keys.to_a - ALL_FIELDS.map & :to_sym
```

Comment:
think this needs to be `ALL_FIELDS.map(&:to_sym)`

Reply:
Thanks, made the change.